core/store/Cargo.toml

```diff
@@ -29,6 +29,7 @@ smart-default = "0.6"
 
 near-crypto = { path = "../crypto" }
 near-primitives = { path = "../primitives" }
+near-chain = { path = "../../chain/chain" }
```

Comment:
Hm, does it actually work? I thought we already have a dep in the other direction?

Reply:
@matklad Nope. I don't know whether I have to re-declare the struct `PartId` again in `store` package or I thought of importing the `chain` module but I get the error of cycle dependency as you can see in the CI . 

So do I re-declare struct in the `store` package or do something else ?

Reply:
You can put the struct to `core/primitives/src/`, that should solve the dependency issue. It's probably best to create a file there like `state_part.rs` and put `PardId` there.